catalog-info.yaml

```diff
@@ -0,0 +1,28 @@
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-fleet-server
+  description: Buildkite Pipeline for fleet-server
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/fleet-server
+
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: fleet-server
+    spec:
+      repository: elastic/fleet-server
+      pipeline_file: ".buildkite/pipeline.yml"
+      teams:
+        ingest-fp:
+          access_level: MANAGE_BUILD_AND_READ

```

Comment:
As in this https://github.com/elastic/package-registry/pull/992#discussion_r1173720764

Maybe inges-tfp should have just "BUILD_AND_READ"

> BUILD_AND_READ: can view and create builds but cannot edit pipeline settings.

But some team should have the permissions MANAGE_BUILD_AND_READ just in case @alexsapran WDYT?

Reply:
I think that the ingest-fp should be able to manage and build, but I don't see a reason why we should lock down the job.

Reply:
Let's keep for the time being MANAGE_BUILD_AND_READ for ingest-fp team. If it is needed, I think it could be updated later